### PR TITLE
Fixed typo and added some extra nested virtualization doc.

### PR DESCRIPTION
**Pull request by teclator**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,9 +27,22 @@ Alternatively, you can just use VirtualBox (no hardware virtualization support
...
-or `host-passthrough`. You can find more information in the [official
+scenario to work you must enable _nested virtualization_ and also expose
+virtualization extensions. 
+
+You can check if the parameter for nested virtualization is enabled with:
+
+    $ cat /sys/module/kvm_intel/parameters/nested #(Intel based machines)
+    $ cat /sys/module/kvm_amd/parameters/nested #(AMD based machines)
+
+And to enable, just add this options to the modules and then reload them or
```
Comment (imobachgs) [inline README.md]: I would say *To enable it* because the next paragraph starts with *And* too.

File: README.md [hunk 2]
```diff
@@ -40,26 +53,26 @@ documentation](https://libvirt.org/formatdomain.html#elementsCPU).
...
-        echo '<username> ALL=NOPASSWD: ALL' >> /etc/sudoers
+         echo '<username> ALL=NOPASSWD: ALL' >> /etc/sudoers
 
   2. Add [YaST:Head](http://download.opensuse.org/repositories/YaST:/Head/openSUSE_42.1/YaST:Head.repo)
      and [devel:languages:ruby:extensions](https://build.opensuse.org/project/show/devel:languages:ruby:extensions)
      repositories. For example, if you're running openSUSE Leap 42.1:
 
-        $ sudo zypper ar -f -r http://download.opensuse.org/repositories/YaST:/Head/openSUSE_42.1/YaST:Head.repo
-        $ sudo zypper ar -f -r http://download.opensuse.org/repositories/devel:/languages:/ruby:/extensions/openSUSE_Leap_42.1/devel:languages:ruby:extensions.repo
+         $ sudo zypper ar -f -r http://download.opensuse.org/repositories/YaST:/Head/openSUSE_42.1/YaST:Head.repo
```
Comment (imobachgs) [inline README.md]: np: Maybe we should consider changing `42.1` to `42.2`. What do you think?

File: lib/aytests/installer.rb [hunk 3]
```diff
@@ -135,7 +135,7 @@ def install_veewee
       File.open(File.join(ENV["HOME"], ".profile"), "a") do |f|
         f.puts "export PATH=#{File.join(Gem.user_dir, "bin")}:\$PATH"
       end
-      install_gem("progressvar", version: PROGRESSBAR_VERSION)
+      install_gem("progressbar", version: PROGRESSBAR_VERSION)
```
Comment (imobachgs) [inline lib/aytests/installer.rb]: 👍

File: README.md (hunk 2 above)
Comment (teclator) [inline README.md]: Yep, if it's working fine with Leap 42.2 then we should update it.

**Pull request merged**